operator/test/test.sh

```diff
@@ -210,7 +211,7 @@ test_results() {
     "call"
     "--channel_creds_type=ssl"
     "--ssl_target=tekton-results-api-service.tekton-results.svc.cluster.local"
-    "--call_creds=access_token=$(kubectl get secrets -n tekton-results -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='tekton-results-debug')].data.token}" | cut -d ' ' -f 2 | base64 --decode)"
+    "--call_creds=access_token=$(kubectl get secrets -n "$NAMESPACE" -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']==\"$RESULTS_SA\")].data.token}" | cut -d ' ' -f 2 | base64 --decode)"
```

Comment:
We have recently changed how we are obtaining tokens, this method may or may not work in future interactions. 
```suggestion
    "--call_creds=access_token=$(kubectl create token $RESULTS_SA -n $NAMESPACE)"
```